src/components/application_manager/include/application_manager/resumption/resume_ctrl.h

```diff
@@ -113,9 +111,9 @@ class ResumeCtrl {
    * @brief RestoreAppWidgets add widgets for the application
    * @param application application which will be resumed
    * @param saved_app application specific section from backup file
-   * @return true if widgets resumed successfully otherwise - false
+   * @return the number of widget windows to be resumed
    */
-  virtual void RestoreAppWidgets(
+  virtual size_t RestoreAppWidgets(
```

Comment:
@YarikMamykin I believe this is not related to wrapper fix

Reply:
@AKalinich-Luxoft I agree with you. This present here because develop branch currently doesn't have 2eafae9d0ae0b352d3b30bd1409bebdcfc447988